Enhance LightBurn Quick Guide with setup and safety tips (#82)
synshop/rtfm: docs/Software/Lightburn Quick Guide.md

LightBurn is a powerful design and control software for laser cutters. Whether you're engraving wood, cutting acrylic, or marking metal, this guide will help you launch your first project with confidence.

## 🔌 Connectors

>   - YOU WILL NEED A USB-A to get files to Lightburn on the dedicated computer for each Laser Cutter.

Reviewer: ```suggestion
  - You'll need to use a USB drive to get files to Lightburn computer
```

>   - Gotcha: there is no USB-A to USB-C adapter in the shop.

Reviewer: ```suggestion
  - If you have a USB-C only USB drive, bring an adapter for USB-A.
```

## 🔌 Setup & Device Connection

> - Turn on the closest computer associated to your **laser device** (dedicated computer). (First Child needs to be turned on on the tiny black box on the wall.)

Reviewer: ```suggestion
- Turn on the computer attached to the laser cutter. 
- First Child's computer is mounted on the wall - find the power button here when using First Child.)
```

> - Launch LightBurn (type 'LightBurn' into the search bar)

Reviewer: ```suggestion
- Launch LightBurn (type "LightBurn" into the search bar)
- Confirm your **workspace size** matches your laser bed dimensions.
```

> - Troubleshooting: Use “Find My Laser” if connected via USB, or manually enter specs for networked machines.

Reviewer: ```suggestion
- If the laser cutter isn't responding to Lightburn,  use "Find My Laser" if connected via USB, or manually enter specs for networked machines.
```

> - Troubleshooting: Confirm your **workspace size** matches your laser bed dimensions.

Reviewer: Let's move this up top as part of the normal process:
```suggestion
```

  - **Console**: View G-code and send manual commands.

## ✏️ Creating Your First Design

> - Use shape tools to draw or import a file (`.SVG`, `.DXF`, `.AI`, `.PNG`). Prefer vectors such as .svg

Reviewer: ```suggestion
- Use shape tools to draw or import a file (`.SVG`, `.DXF`, `.AI`, `.PNG`). Prefer the three  vector based formats vs the `.PNG` which is a raster based format. 
```

- Assign each object to a **layer** and configure settings:

>   - **Speed**: How fast the laser moves. Try: 4 for cardboard, 1 for ply, 0.7 for 1/8 MDF.

Reviewer: These tips about starting values are super helpful - thanks!

  - **Power**: Laser intensity (as a percentage). Try: 15% for cardboard. 40% for 1/8" wood. 60% MDF 1/8".
  - **Mode**: Line (cut), Fill (engrave), or Offset Fill.
- If you don't care about burned-looking edges, you can increase the power a bit, to make sure you make it through the stock.

## 📐 Positioning & Preview
- Suggestion: For Line (cut) mode, try on cardboard first (light medium requires only Speed:4 and Power ~15%)

> - Different printers have different effective areas, even within the working space. For example, on First Child, the light 'Power' may need to be increased to effect cuts in the bottom 1/5. 

Reviewer: This is actively changing and always trying to be fixed, we can give them a warning, but not give advice that might be out of date next week:

```suggestion
- Different printers have different effective areas, even within the working space. Ask a shop host or on Discord if you're seeing uneven cuts across the working space. 
```

- Use the **Frame** button to test that the shapes will fit on the medium in the machine.
- Click **Preview** to simulate the job and estimate time.

## 🚀 Sending the Job
- Double-check your material placement.
- Double-check that the machine is on (some machines need to be 'on' at the Power Strip) and that Lightburn sees it. You might need to restart Lightburn.

> - Double-check that the air is on (it should come on automatically.

Reviewer: ```suggestion
- Double-check that the air is on by putting your hand under the cut head - you should feel a steady stream of blowing air. It comes on automatically, but you can check it if you're seeing irregular results..
```

>   - Don't mess with the wall switch unless you know what you are doing. The wall-switch is 'off by default' when switched to the right. Ask someone if you are unsure.

Reviewer: I found the wording a bit confusing. I think we can be more explicit about how it works and why you normally don't need to change it:

```suggestion
- The fan for all laser cutters is auto detecting: when a laser cutter turns on, the fan should turn on automatically.  You shouldn't need to use the wall switch to turn the fan on or off. 
```

> - Double-check the 'Focal length' using the little orange tool (hang it on the arm). Other projects may have had a thicker project material; you will need to move the bed up or down until the orange arm rests exactly.

Reviewer: ```suggestion
- Double-check the focal length by using the little orange acrylic tool (hang it on the arm). Other projects may have had a thicker project material; you will need to move the bed up or down until the orange arm rests exactly.
```

- Click **Start** to begin the job.
- Monitor progress and pause if needed.

## 🧰 Tips & Safety
- Always wear safety glasses.
- Ventilate your workspace.
- Use scrap material to test settings before final runs.

> - For Line (cut) mode: Look for smoke coming from the underside of the project. This indicates it has successfully cut all the way.

Reviewer: ```suggestion
- When cutting stock with [Line Mode](https://docs.lightburnsoftware.com/2.1/Reference/CutSettingsEditor/LineMode/#line-mode-settings), look for smoke coming from the underside of the project. This indicates it has successfully cut all the way through.
```

> - for Line (cut) mode, cutting twice is fine, as long as you have not moved the stock. Good luck getting it back in position.

Reviewer: ```suggestion
- Re-running a cut twice is fine, but you do not move the stock. It is near impossible to get back in the exact same position after moving it.
```

> - Some machines have other pecadillos. (First child needs a support bar while open)

Reviewer: ```suggestion
- First child sometimes needs a support bar to keep the top lid open 
```

## 🧰 Advanced Considerations

> - Does your Line (cut) project need fiducials? Design time is the best time to add them.

Reviewer: ```suggestion
- Does  project need [fiducials](https://en.wikipedia.org/wiki/Fiducial_marker)? Design time is the best time to add them.
```

> - Does your project have lots of small bits and bobs? Does it need numbering on the back-side of the bits?

Reviewer: ```suggestion
- Does your project have lots of small bits and bobs? Consider adding numbering on the back-side of the bits before you cut.
```

## 🧰 Cleanup and Shutdown
- For First Child, this is pretty straighforward
  - Turn off at the power strip.

>   - Fine to leave your non-useful scrap in the corner.

Reviewer: ```suggestion
- Leave it cleaner than you found it
- Turn off power
- Shop vac any small pieces you see left in laser cutter   
- Unwanted larger scrap can be put in the labeled  "scrap" bin so other members may use it
```
